ocs_ci/templates/CSI/rbd/pod_block.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
this yaml can be moved to ocs_ci/templates/app-pods  because it doesn't look good at this location

Reply:
Done!